### PR TITLE
use sha256

### PR DESCRIPTION
**Pull request by Nesurion**
use sha256 hash instead of the sha1 hash, to fix the homebrew warning.
https://github.com/giantswarm/giantswarm/issues/593

### CODE REVIEW

File: crosscompile.sh [hunk 1]
```diff
@@ -31,20 +31,20 @@ function package_version() {
...
   cat > $FORMULA << EOF
 require "formula"
 
 class SwarmClient < Formula
   homepage "https://giantswarm.io"
-  # openssl dgst -sha1 <file>
+  # openssl dgst -sha256 <file>
   url "https://downloads.giantswarm.io/swarm/clients/$VERSION/$TAR.tar.gz"
-  sha1 "$SHA1"
+  sha256 "$SHA256"
```
Comment (xh3b4sd) [inline crosscompile.sh]: Why are we doing this? Is this the hash used for the homebrew recipe?

Comment (Nesurion) [inline crosscompile.sh]: jep. Brew compares the hash of the tar it downloaded with this hash.

**Comment by xh3b4sd** [comment]
This LGTM. 

I only wonder if we want to allign the release procedure with the other open source repos. We are not using the github releases for the swarm cli.

**Comment by Nesurion** [comment]
@xh3b4sd true. I'll create an issue for that and merge this for now.

**Comment by JosephSalisbury** [comment]
As a note, we're using both github releases and S3 currently (we just added the github releases support). Saying that, moving it to GitHub is cool.

**Pull request merged**